packages/makeup-floating-label/src/index.js

```diff
@@ -44,6 +52,23 @@ module.exports = class {
         if (!hasValue(this.textboxEl) && !isAutofilled(this.textboxEl, this.options.textboxElementBackgroundRGB)) {
             this.labelEl.classList.add(this.options.labelElementInlineModifier);
         }
+        if (document.activeElement === this.textboxEl) {
+            this.labelEl.classList.add(this.options.labelElementFocusModifier);
+        }
+
+        if (isInvalid(this.textboxEl)) {
+            this.labelEl.classList.add(this.options.labelElementInlineModifier);
+        }
+    }
+
+    setInvalid(hasError) {
+        if (hasError) {
+            this.textboxEl.setAttribute('aria-invalid', 'true');
+            this.labelEl.classList.add(this.options.labelElementInlineModifier);
```

Comment:
Should this be `labelElementInvalidModifier` instead of `labelElementInlineModifier`? Same goes for line 70.